sourcemash/frontend/static/js/routers/app_router.js

```diff
@@ -6,6 +6,7 @@ Sourcemash.Routers.AppRouter = Backbone.Router.extend({
         "categories/:id": "showCategory",
         "saved": "showSaved",
         "browse": "browseFeeds",
+        "categorizer": "showCategorizer",
```

Comment:
Or accept that it’s a separate feature that those who know about it can access by URL or the back button?

-- 
Alex Gerstein
gerstein.alex@gmail.com

On May 24, 2015 at 10:30:04 PM, Scott Gladstone (notifications@github.com) wrote:

In sourcemash/frontend/static/js/routers/app_router.js:

> @@ -6,6 +6,7 @@ Sourcemash.Routers.AppRouter = Backbone.Router.extend({
>          "categories/:id": "showCategory",
>          "saved": "showSaved",
>          "browse": "browseFeeds",
> +        "categorizer": "showCategorizer",
Here are the (eh) options as i see them: 1. Side nav: fine but its getting cluttered. 2. Top nav: eh, don't love it 3. Turn help menu button into a toolbox and put the help menu and categorizer in that: also don't love it 4. "Hide it" by making it link from only one page (eg splash) and the help menu: worst
—
Reply to this email directly or view it on GitHub.